server/src/test/java/io/crate/geo/GeoJSONUtilsTest.java

```diff
@@ -125,22 +121,23 @@ public void testMapFromWktRoundTrip() throws Exception {
         Map<String, Object> map = GeoJSONUtils.shape2Map(shape);
 
         Map<String, Object> wktMap = GeoJSONUtils.wkt2Map(wkt);
-        assertThat(map.get("type"), is(wktMap.get("type")));
-        assertThat(map.get("coordinates"), is(wktMap.get("coordinates")));
+        assertThat(map.get("type")).isEqualTo(wktMap.get("type"));
+        assertThat(map.get("coordinates")).isEqualTo(wktMap.get("coordinates"));
 
         Shape mappedShape = GeoJSONUtils.map2Shape(map);
         String wktFromMap = new WKTWriter().toString(mappedShape);
-        assertThat(wktFromMap, is(wkt));
+        assertThat(wktFromMap).isEqualTo(wkt);
     }
 
     @Test
     public void testInvalidWKT() throws Exception {
-        assertThrows(IllegalArgumentException.class,
-                     () -> GeoJSONUtils.wkt2Map(
-                         "multilinestring (((10.05  10.28  3.4  8.4, 20.95  20.89  4.5  9.5),\n" +
-                         " \n" +
-                         "( 20.95  20.89  4.5  9.5, 31.92  21.45  3.6  8.6)))"),
-                     "Cannot convert WKT \"multilinestring (((10.05  10.28  3.4  8.4, 20.95  20.89  4.5  9.5)");
+        assertThatThrownBy(
+            () -> GeoJSONUtils.wkt2Map(
+                "multilinestring (((10.05  10.28  3.4  8.4, 20.95  20.89  4.5  9.5),\n" +
+                    " \n" +
+                    "( 20.95  20.89  4.5  9.5, 31.92  21.45  3.6  8.6)))"),
+            "Cannot convert WKT \"multilinestring (((10.05  10.28  3.4  8.4, 20.95  20.89  4.5  9.5)")
```

Comment:
shouldn't this be `.hasMessageStartingWith("Cannot convert WKT \"multilinestring (((10.05  10.28  3.4  8.4, 20.95  20.89  4.5  9.5)");`?